nuget/Microsoft.Windows.CppWinRT.targets

```diff
@@ -590,4 +590,37 @@ $(XamlMetaDataProviderPch)
         </Link>
     </ItemDefinitionGroup>
 
+    <!--
+    ============================================================
+    Override the CopyWinMDArtifactsOutputGroup to return the 
+    correct WinMD. Not that we still need to call GetResolvedWinMD
+    to make sure that projects that only use CppWinRT to consume
+    WinRT, but not to produce the WinMD, will still return the 
+    WinMD file produced by the CX target GetResolvedWinMD.
+    ============================================================
+    -->
+
+    <Target Name="WinMDArtifactsProjectOutputGroup"
```

Comment:
what's the purpose of this empty target?

Reply:
It is. WIll add a comment.

Reply:
Yup, this is a standard output group target used for WinMDs and known to VS. Added someclarification.